### PR TITLE
updated homepage

### PR DESCRIPTION
**Pull request by jeffding412**
added banner image and story corp + parallax

### CODE REVIEW

File: client/src/pages/Homepage/Homepage.js [hunk 1]
```diff
@@ -4,30 +4,27 @@ import './Homepage.css';
 class Homepage extends Component {
   render() {
     return (
-      <div id="content" className="container-fluid">
+      <div id="content">
         <div className="row">
```
Comment (chauandrew) [inline client/src/pages/Homepage/Homepage.js:8]: Don't think this class is necessary. Noticed that it's causing the page to use a horizontal scroll bar

File: client/src/pages/Homepage/Homepage.js [hunk 2]
```diff
@@ -4,30 +4,27 @@ import './Homepage.css';
...
           <div className="col-md-5 align-self-center" id="nswnIs">
```
Comment (chauandrew) [inline client/src/pages/Homepage/Homepage.js:9]: Could we offset this by 1 column and use a width of 4 columns?

File: client/src/pages/Homepage/Homepage.css [hunk 3]
```diff
@@ -2,23 +2,70 @@ html, body {
...
 }
 
+.container-fluid {
+    padding: 80px 120px;
+}
+
 #content {
     height: 100%;
     width: 100%;
     background: linear-gradient(90deg, rgba(255, 255, 255, 0.84) 47%, #7799A4 53%);
```
Comment (chauandrew) [inline client/src/pages/Homepage/Homepage.css:12]: Can we move the background from the page content to just the first div?

File: client/src/pages/Homepage/Homepage.js [hunk 4]
```diff
@@ -4,30 +4,27 @@ import './Homepage.css';
...
-      //   </header>
-      //   <p className="App-intro">{this.state.apiResponse}</p>
-      // </div>
+
+        <div class="container-fluid parallax">
+          <div className="text-center" id="imageText">
+            <h1>I am John Doe</h1>
+            <p>And I'm a Photographer</p>
+          </div>
+        </div>
```
Comment (chauandrew) [inline client/src/pages/Homepage/Homepage.js:20]: Can we create a separate parallax component and then import that? Figured we might want to add it on other pages.

File: client/src/pages/Homepage/Homepage.js [hunk 5]
```diff
@@ -4,30 +4,27 @@ import './Homepage.css';
...
+
+        <div id="pizza" className="text-center">
```
Comment (chauandrew) [inline client/src/pages/Homepage/Homepage.js:22]: I like pizza

File: client/src/pages/Homepage/Homepage.css (hunk 3 above)
Comment (YiyueMaggieMao) [reply client/src/pages/Homepage/Homepage.css:12]: Minor suggestion: We can add ``` min-width: 100vw ``` to the parallax class to get rid of the white space on the right hand side of the div

**Pull request merged**